changes/5158.housekeeping

```diff
@@ -0,0 +1 @@
+Fixed `nautobot.core.tests.test_views.ErrorPagesTestCase.(test_500_default_support_message, test_500_custom_support_message)` failing unittest.
```

Comment:
Let's remove this fragment since the failure was only seen in the prototype branch.